src/editors/containers/ProblemEditor/components/SelectTypeModal/content/Preview.jsx

```diff
@@ -1,37 +1,63 @@
 import React from 'react';
+import { connect } from 'react-redux';
 import PropTypes from 'prop-types';
+import { Hyperlink, Image } from '@edx/paragon';
+import {
+  FormattedMessage,
+  injectIntl,
+  intlShape,
+} from '@edx/frontend-platform/i18n';
+import messages from './messages';
+import { selectors } from '../../../../../data/redux';
 import { ProblemTypes } from '../../../../../data/constants/problem';
 
-const Preview = ({
+export const Preview = ({
+  // redux
   problemType,
```

Comment:
ah, so this will be wrong, as this preview should show the local `selected` value, not the problemType. This will make it so that the user sees the preview for the item they have selected. Having the redux value here will not work